packages/react-components/react-shared-contexts/src/ProviderContext/ProviderContext.ts

```diff
@@ -1,11 +1,20 @@
 import * as React from 'react';
-import type { ProviderContextValue } from './ProviderContext.types';
 
-export const ProviderContext = React.createContext<ProviderContextValue>({
+export type ProviderContextValue = {
```

Comment:
Nit: Did we need to get rid of the types.ts file? We usually separate types into their own files.